arch/arm64/boot/dts/qcom/sm7325.dtsi

```diff
@@ -15,3 +17,102 @@
 &cpu5 { compatible = "qcom,kryo670"; };
 &cpu6 { compatible = "qcom,kryo670"; };
 &cpu7 { compatible = "qcom,kryo670"; };
+
+&lpass_audiocc {
+	compatible = "qcom,sm7325-lpassaudiocc",
+		     "qcom,qcm6490-lpassaudiocc";
+	/delete-property/ power-domains;
+ };
```

Comment:
I've fixed this with https://lore.kernel.org/linux-arm-msm/20250930-sc7280-dts-misc-v1-5-5a45923ef705@fairphone.com/, can drop this from sm7325.dtsi